src/App.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  if (true) {
+    // Error on purpose
+    useState();
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
React Hook "useState" is called conditionally. React Hooks must be called in the exact same order in every component render. [react-hooks/rules-of-hooks](https://eslint.org/docs/rules/react-hooks/rules-of-hooks)